jax/lib/__init__.py

```diff
@@ -51,7 +51,7 @@ def _check_jaxlib_version():
 
 from jaxlib import xla_client
 from jaxlib import lapack
-from jaxlib import pytree
```

Comment:
Because the C++ and Python parts of JAX are released separately, you'll either need to make this change conditional on the jaxlib version, or make it another PR after a jaxlib release and minimum version bump. I would probably just make it conditional on the jaxlib version.

Reply:
Trying this new approach.